### PR TITLE
fix(registry-change): paginate db requests

### PR DESCRIPTION
**Pull request by hulkoba**

### CODE REVIEW

File: jobs/registry-change.js [hunk 1]
```diff
@@ -157,10 +157,7 @@ module.exports = async function (
   // get config
   const keysToFindMonorepoDocs = _.compact(_.map(withMultiplePackageJSON, (group) => group[0].value.fullName.toLowerCase()))
   if (keysToFindMonorepoDocs.length) {
-    const monorepoDocs = (await repositories.query('by_full_name', {
```
Comment (janl) [inline jobs/registry-change.js]: just one note: here we do `repositories.query()` and further down, you‘re replacing it with a call to `getAllDocs()` which does `repositories.allDocs()`, which is not the same :)

Comment (hulkoba) [reply jobs/registry-change.js]: I renamed it to 'queryDocs' for less confusion :)

Comment (janl) [reply jobs/registry-change.js]: thanks!

File: test/jobs/registry-change.js [hunk 2]
```diff
@@ -615,6 +650,35 @@ describe('monorepo-release: registry change create jobs', async () => {
...
+        '2.0.0': {
+          gitHead: 'smurf'
+        },
+        '1.0.0': {
+          gitHead: 'sky'
+        }
+      },
+      registry: 'https://skimdb.npmjs.com/registry'
+    })
+    expect(newJobs).toHaveLength(0)
```
Comment (hulkoba) [inline test/jobs/registry-change.js]: here

Comment (janl) [reply test/jobs/registry-change.js]: maybe there should be jobs

Comment (hulkoba) [reply test/jobs/registry-change.js]: There are jobs now

**Pull request merged**